script.js

```diff
@@ -222,24 +222,53 @@ function showMangaDetails(manga, categories, source) {
     document.getElementById('manga-source').textContent = source;
     const mangaThumbnail = document.getElementById('manga-thumbnail');
     mangaThumbnail.src = manga.customThumbnailUrl || manga.thumbnailUrl;
-    const mangaStatus = (() => {
+    document.documentElement.style.setProperty('--manga-header-bg', `url('${mangaThumbnail.src}')`);
+    const mangaAuthor = document.getElementById('manga-author');
+    const mangaArtist = document.getElementById('manga-artist');
+    const mangaDescription = document.getElementById('manga-description');
+    const mangaStatus = document.getElementById('manga-status');
+
+    mangaStatus.innerHTML = '';
+    const mangaStatusText = (() => {
       switch (manga.status) {
-          case 1: return 'Ongoing';
-          case 2: return 'Completed';
-          case 3: return 'Licenced';
-          case 4: return 'Publishing Finished';
-          case 5: return 'Cancelled';
-          case 6: return 'On Hiatus';
-          default: return 'Unknown';
+          case 1: 
+	    addMaterialSymbol(mangaStatus, 'schedule');
+	    return 'Ongoing';
+          case 2:
+	    addMaterialSymbol(mangaStatus, 'done_all');
+	    return 'Completed';
+	  case 3:
+	    addMaterialSymbol(mangaStatus, 'attach_money');
+	    return 'Licenced';
+          case 4: 
+	    addMaterialSymbol(mangaStatus, 'check');
+	    return 'Publishing Finished';
+          case 5:
+	    addMaterialSymbol(mangaStatus, 'close');
+	    return 'Cancelled';
+          case 6:
+	    addMaterialSymbol(mangaStatus, 'pause');
+	    return 'On Hiatus';
+          default:
+	    addMaterialSymbol(mangaStatus, 'block');
+	    return 'Unknown';
```

Comment:
I made a test JSON just to be able to see those icons xD
[Status-Demo.json](https://github.com/user-attachments/files/15872565/Status-Demo.json)